Makefile

```
@@ -59,6 +59,9 @@ endif
 julia-deps: | $(DIRS) $(build_datarootdir)/julia/base $(build_datarootdir)/julia/test $(build_defaultpkgdir)
 	@$(MAKE) $(QUIET_MAKE) -C $(BUILDROOT)/deps
 
+julia-stdlib:
```

```suggestion
julia-stdlib: | $(DIRS)
```

Also need to be added in line 12
```Makefile
 BUILDDIRS := $(BUILDROOT) $(addprefix $(BUILDROOT)/,base src ui doc deps test test/embedding test/llvmpasses)
```